### PR TITLE
javacOptions in Compile configuration (not compile task)

### PR DESCRIPTION
**Pull request by chbatey**
So they can be added to in multiple places

### CODE REVIEW

File: project/JdkOptions.scala [hunk 1]
```diff
@@ -40,33 +44,41 @@ object JdkOptions extends AutoPlugin {
       // EnvelopeBuffer.class with 'javap -c': it should refer to
       //""java/nio/ByteBuffer.clear:()Ljava/nio/Buffer" and not
       // "java/nio/ByteBuffer.clear:()Ljava/nio/ByteBuffer". Issue #27079
-      (java8home: File) => Seq("-release", "8", "-javabootclasspath", java8home + "/jre/lib/rt.jar")
-    )
-  def targetJdkJavacOptions(targetSystemJdk: Boolean, jdk8home: Option[File], fullJavaHomes: Map[String, File]): Seq[String] =
+      (java8home: File) => Seq("-release", "8", "-javabootclasspath", java8home + "/jre/lib/rt.jar"))
+    println("Selecting options: " + options)
```
Comment (jrudolph) [inline project/JdkOptions.scala:48]: left over?

**Review by jrudolph — APPROVED** [review]
LGTM (println left over as noted)

File: project/JdkOptions.scala (hunk 1 above)
Comment (chbatey) [reply project/JdkOptions.scala:48]: Heh yep

**Comment by jrudolph** [comment]
Akka HTTP nightlies were also affected: https://jenkins.akka.io:8498/view/Akka-http/job/akka-http-nightly/216/AKKA_VERSION=master,Node=akka-http-nightly-node,SCALA_VERSION=2.12.11,jdk=AdoptOpenJDK%208/

**Comment by chbatey** [comment]
Seems this breaks javdoc, looking into it

**Review by ennru — APPROVED** [review]
LGTM.

**Pull request merged**